test/deploy_test.rb

```diff
@@ -48,16 +48,16 @@ def check_process(pid)
         raise e
       end
     end
-    it "should start/stop sidekiq" do
-      sidekiq_pid_path = @env_root.join("deploy", "shared", "pids", "sidekiq.pid")
-      # fresh deploy
-      mina "deploy"
-      mina "sidekiq:start"
-      pid = File.read(sidekiq_pid_path)
-      check_process(pid).must_equal true
-      # second deploy
-      mina "sidekiq:stop"
-      check_process(pid).must_equal false
-    end
+    #it "should start/stop sidekiq" do
+    #  sidekiq_pid_path = @env_root.join("deploy", "shared", "pids", "sidekiq.pid")
```

Comment:
Sorry, this isn't ready to merge as is. I've got an issue open over on the mina repo about this problem. I'll update before the mina release. I was actually changing this branch so I could use it locally - I probably should have a different branch for the PR!

https://github.com/mina-deploy/mina/issues/414